src/graphic.rs

```diff
@@ -168,27 +168,27 @@ pub fn tilemap_coords_px(_tilesize: u32, graphic: Graphic) -> Option<(i32, i32)>
         FoodAcornWide => Some((2 + 3, 12 - 3)),
         FoodAcornThin => Some((3 + 3, 12 - 3)),
         FoodCarrotWide => Some((1 + 3, 12 - 3)),
-        FoodCarrotSideways => Some((0 + 3, 12 - 3)),
+        FoodCarrotSideways => Some((3, 12 - 3)),
```

Comment:
I know it's weird but I'd prefer to keep this as is.

These numbers are offsets in the tilemap position and as such I like them to keep an explicit format that's the same for every tile.

Is there a way to allow this particular warning at a block/function level?